src/Components/Product/Filter.js

```diff
@@ -0,0 +1,67 @@
+function Filter() {
+  return (
+    <div className="filter-container">
+                    <div className="filter-head flex-row align-center justify-bw">
+                        <h4>Filter</h4>
+                        <a className="text-dec-none font-primary-color" href="#top">Clear</a>
+                    </div>
+                    <div className="filter-price">
+                        <h4 className="margin-tb-xs">Price</h4>
+                        <div className="flex-row justify-bw">
+                            <span>50</span>
+                            <span>100</span>
+                            <span>200</span>
+                        </div>
+                        <div className="slidecontainer">
+                            <input className="slider" type="range" min="1" max="100" value="50" />
```

Comment:
suggestion: use defaultValue instead of value

Reply:
Thanks for this, just found out this bug.
Resolved now.